src/main/java/org/apache/sling/feature/cpconverter/handlers/IndexDefinitionsEntryHandler.java

```diff
@@ -126,8 +126,9 @@ public void handle(@NotNull String path, @NotNull Archive archive, @NotNull Entr
 
                 } else {
                     // binary file, should we attach?
-                    if ( archive.getMetaInf().getFilter().contains(repositoryPath)) {
-                        indexManager.getIndexes().registerBinary(repositoryPath, is);
+                    String binaryParentPath  = repositoryPath.substring(0,repositoryPath.lastIndexOf("/")) ;
```

Comment:
I don't think this is how FileVault works, you really need to check if the node path given through repository path is contained in the filter (otherwise it won't be imported). Just think about more complicated use cases like excludes below a specific filter root.